src/routes/applicant.ts

```diff
@@ -1,11 +1,14 @@
 import * as express from 'express';
-import { createApplicant, updateApplicantStatus, listAllApplicants } from '../controllers/applicant';
+import { createApplicant, updateApplicantStatus, listAllApplicants, findGrade } from '../controllers/applicant';
 
 const router = express.Router();
 
 /* Returns a list of all applicants currently in the database */
 router.get('/', listAllApplicants);
 
+/* Return the grade of an applicant given the id passed into the request body */
+router.post('/grade/', findGrade);
```

Comment:
This is a get route, not post. Post is for creating new objects

I would change this API call to `router.post('/grade/:applicantId', findGrade)` in this case and not place it in the body.